### PR TITLE
Unit Tests about keywords and unit test step in CI

### PR DESCRIPTION
**Pull request by NicoMincuzzi**
* Add TEST_REQUIRE in `setup.py`
* Add unit tests about `keyword.py` in `utests` folder
* Add `Test with pytest` step in CI

### CODE REVIEW

File: ArchiveLibrary/keywords.py [hunk 1]
```diff
@@ -111,20 +116,14 @@ def create_zip_from_files_in_directory(self, directory, filename, sub_directorie
...
-            comp_method = zipfile.ZIP_BZIP2
-        elif compression == "lzma":
-            comp_method = zipfile.ZIP_LZMA
-        else:
-            raise ValueError ("Unknown compression method")
-        
+
+        if cls.compressions.get(compression) is None:
+            raise ValueError("Unknown compression method")
+        comp_method = cls.compressions.get(compression)
```
Comment (lucagiove) [inline ArchiveLibrary/keywords.py:122]: :clap:

File: ArchiveLibrary/keywords.py [hunk 2]
```diff
@@ -93,14 +97,15 @@ def create_tar_from_files_in_directory(self, directory, filename, sub_directorie
...
-        files = return_files_lists(directory, sub_directories)
 
+        files = return_files_lists(directory, sub_directories)
         for filepath, name in files:
             tar.add(filepath, arcname=name)
 
         tar.close()
 
-    def create_zip_from_files_in_directory(self, directory, filename, sub_directories=False, compression="stored"):
+    @classmethod
```
Comment (lucagiove) [inline ArchiveLibrary/keywords.py:107]: Why the classmethod?

File: setup.py [hunk 3]
```diff
@@ -1,20 +1,21 @@
 #!/usr/bin/env python
-
-from distutils.core import setup
+import sys
 from os.path import abspath, dirname, join
 
+from setuptools import setup
```
Comment (lucagiove) [inline setup.py:5]: :+1:

**Review by lucagiove — APPROVED** [review]

File: ArchiveLibrary/keywords.py (hunk 2 above)
Comment (NicoMincuzzi) [reply ArchiveLibrary/keywords.py:107]: In this case create_zip_from_files_in_directory doesn't use `self` and it needs of `compression` dict.

**Pull request merged**